src/Protocol/PacketID.cpp

```diff
@@ -0,0 +1,182 @@
+#include "Globals.h"
+
+#include "Protocol_1_9.h"
+#include "Protocol_1_12.h"
+
+
+
+
+
+UInt32 cProtocol_1_9_0::GetPacketId(eOutgoingPackets a_Packet)
+{
+	switch (a_Packet)
+	{
+	case sendAttachEntity:          return 0x40;
```

Comment:
Style: all the `case`s should have one more indent (the curly brace controls the indent)

Reply:
Fixed.